.travis.yml

```diff
@@ -1,12 +1,59 @@
+conditions: v1
+dist: trusty
+sudo: false
 language: python
+
 python:
-  - "2.6"
-  - "2.7"
-  - "3.2"
-  - "3.3"
-  - "3.4"
-  - "3.5"
-  - "3.5-dev" # 3.5 development branch
-  - "nightly" # currently points to 3.6-dev
-# command to run tests
+- 3.6
+
 script: nosetests
+
+jobs:
+  include:
+  - python: 3.7
+    dist: xenial
+    sudo: true
+
+  - python: 3.5
```

Comment:
All Pythons except for 3.7 don't need sudo, nor they need xenial. Revert this change and only leave 3.7 inside of jobs.include